main.go

```diff
@@ -0,0 +1,24 @@
+/*
+ *Author: Stefan
+ *Date: 11/20/2019
+ *Last changes: 11/20/2019 12.50
+ *Task: Implement average([]int) float64 function that returns an average value of slice (sum / N)
+**/
+
+package main
+
+import "fmt"
+
+func average(sl []int) float64 {
+	var N int = len(sl)
+	var sum int = 0
```

Comment:
can be simplified to `sum := 0` or `var sum int`

Reply:
Great, thanks! Done)